### ISSUE TITLE
When I wire points to someone, the notification icon is their icon, not mine. "You have successfully wired"

### ISSUE THREAD

**Issue by xftroxgpx** [Minds/front#32]
**tl;dr:** the notification that I get, after wiring points to someone, has their icon not mine .
But when they wire to me, it still has they icon (which is probably correct).
This also means that when **I** wire to someone, I may not get the notification, because it's considered a notification from them.

![screenshot_2017-10-05_01-28-42](https://user-images.githubusercontent.com/26243389/31204617-d718a5de-a96c-11e7-9416-2640c3bb6858.png)

-----

When I wire points to someone, the notification that I get has their icon (red in screenshot)
and because of this, it seems, that I don't receive these notifications when I wire to users that Muted/Blocked me?(or some else is going on)

![screenshot_2017-10-05_01-01-52](https://user-images.githubusercontent.com/26243389/31204246-4032ab2a-a96b-11e7-9850-9e071c62e797.png)

When I wire to myself, the two notifications are the same:
If I wire some points to myself, [this](https://github.com/Minds/front/commit/9c758af04ac83701433c051ca9430501f215fbd3#diff-c0c6944a0c5b5ce61749bd2fd0121c0cR470) ("You have successfully wired...to...") is the message that gets triggered, twice. Instead of, once this and once the next one with "You have received a wire... from ...". This used to work prior to [Aug 28th](https://www.minds.com/media/748994940473712655)

Does this mean `notification.params.from_guid == session.getLoggedInUser().guid` is true for both notifications?

I think the `notification.params.from_guid` is the wired-to user's guid for both notifications, instead of mine. And since [that commit](https://github.com/Minds/front/commit/9c758af04ac83701433c051ca9430501f215fbd3#diff-c0c6944a0c5b5ce61749bd2fd0121c0cR470) is the latest for that change and it was 2 months ago, something else changed (not that html) which made this issue possible.

![screenshot_2017-10-05_00-41-58](https://user-images.githubusercontent.com/26243389/31203225-6d31e758-a966-11e7-9ee0-9b626677e6fb.png)

When I receive a wire, it's still the sender's icon that's showing (which is probably good?)
![screenshot_2017-10-05_01-25-17](https://user-images.githubusercontent.com/26243389/31204452-270870d4-a96c-11e7-8084-451b51efcba9.png)

**EDIT1:** on mobile the notification that they sent points to me shows an error (and notice how it has my icon not his) (according to pinn's comment in [this](https://www.minds.com/newsfeed/762443547273076736) thread)
![1](https://user-images.githubusercontent.com/26243389/31204833-0ec8e4a2-a96e-11e7-90ba-84ce0a31726e.jpg)
